### ISSUE TITLE
Cannot run ejabberd with ejabberd/mix image

### ISSUE THREAD

**Issue by mremond** [processone/docker-ejabberd#30]
I have been using ejabberd cloned from git.

I then would build it properly with the commands described in README

```
git clone https://github.com/processone/ejabberd.git
cd ejabberd
docker run --rm -v $(pwd):$(pwd) -w $(pwd) ejabberd/mix do deps.get, deps.compile, compile
```

However, the command to make it run fails:

```
docker run --rm -it -p 5222:5222 -p 5280:5280 -v $(pwd):$(pwd) -w $(pwd) --entrypoint="/usr/bin/iex" ejabberd/mix -S mix
```

It fails due to missing configuration file:
```
15:24:09.020 [error] Cannot load /Users/mremond/devel/ejabberd/ejabberd/config/ejabberd.yml: no such file or directory

=INFO REPORT==== 26-Dec-2018::15:24:09 ===
    application: lager
    exited: stopped
    type: temporary
[os_mon] cpu supervisor port (cpu_sup): Erlang has closed
[os_mon] memory supervisor port (memsup): Erlang has closed
```

I had to do:

```
cp ejabberd.yml.example config/ejabberd.yml
```

**Comment by cromain** [comment]
this file was removed as it produced some confusion about which yml is "the good one"
our Dockerfile copy it's correct ejabberd.yml from docker-ejabberd repo in order to make it run.
we likely need to improve the readme for those who compile ejabberd themself using ejabberd/mix, adn maybe add a simple wrapper script which will cope with all environement setup for simpler deployment.

**Comment by cromain** [comment]
I fixed README to include to copy procedure in 958ab84
